pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -164,6 +165,9 @@ func (pl *InterPodAffinity) PreScore(
 		podInfo:       framework.NewPodInfo(pod),
 	}
 
+	// There should be at most len(allNodes) topology maps produced.
+	topoScores := make([]scoreMap, len(allNodes), len(allNodes))
```

Comment:
nit: this is equivalent with `topoScores := make([]scoreMap, len(allNodes))`, not quite sure which way is more conventional.

Reply:
As anything in go: the one with less characters :)